src/Microsoft.PowerShell.Commands.Management/Microsoft.PowerShell.Commands.Management.csproj

```diff
@@ -1,11 +1,23 @@
-﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
+﻿<Project ToolsVersion="15.0">
+
   <Import Project="..\..\PowerShell.Common.props" />
   <PropertyGroup>
     <Description>PowerShell's Microsoft.PowerShell.Commands.Management project</Description>
     <NoWarn>$(NoWarn);CS1570</NoWarn>
     <AssemblyName>Microsoft.PowerShell.Commands.Management</AssemblyName>
   </PropertyGroup>
 
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" Condition="'$(SDKToUse)' != 'Microsoft.NET.Sdk.WindowsDesktop' " />
```

Comment:
It seems we could remove condition:
```c#
  <Import Project="Sdk.props" Sdk="$(SDKToUse)" />
```

Reply:
I suppose we could simplify that and update PSxUnit to define `SDKtoUse`

Reply:
Unfortunately, the way typegen is built and run, wasn't able to define this property.  So keeping the current line.

Reply:
It is not clear if we define SDKToUse in Build.psm1 and can use the property in the condition why we can not use it in the assignment.

Reply:
I tried a few things to pass SDKToUse, but can't get it working with typegen (the constant isn't being propagated to the imported csproj from Microsoft.PowerShell.SDK.csproj.  I think we can do this as a separate PR if you want to look into it.

Reply:
Do mean "Start-PSBuild -TypeGen" ?

Reply:
The typegen code itself (run when either `-TypeGen` or `-Clean` is used) is built and run differently.  It seems that when it imports other csproj, the SDKToUse doesn't propagate which causes the build to fail

Reply:
Thanks for clarify! After we merge the PR I hope to have a time to look this puzzle :-)